wrappa/api_auth_wrappa.go

```diff
@@ -140,12 +142,17 @@ func (wrappa *APIAuthWrappa) Wrap(handlers rata.Handlers) rata.Handlers {
 			panic("you missed a spot")
 		}
 
-		if name == atc.GetAuthToken {
+		if name == atc.GetAuthToken || name == atc.GetAccessToken {
 			newHandler = auth.WrapHandler(newHandler, wrappa.getTokenValidator, wrappa.userContextReader)
 		} else {
 			newHandler = auth.WrapHandler(newHandler, wrappa.authValidator, wrappa.userContextReader)
 		}
-		wrapped[name] = auth.CSRFValidationHandler(newHandler, rejector, wrappa.userContextReader)
+		if name != atc.CheckResourceWebHook {
```

Comment:
We were expecting this question!  @mainephd can probably provide more context, but:

`CSRFValidationHandler` expects the client to be authenticated for all GET API calls (e.g. DownloadCLI, ListAllPipelines, etc).  Issue here is that the webhook is a POST, and won't get the exception inside `CSRFValidationHandler`, hence this check.